src/index.html

```diff
@@ -17,6 +29,65 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="menu">
+      <a href="_blank">
+        <img
+          src="images/logo.png"
+          alt="Moyo Logo"
+          class="logo"
+        />
+      </a>
+      <nav class="nav">
+        <a
+          href="_blank"
+          class="link is-active"
+        >
+          apple
+        </a>
+        <a
+          href="_blank"
+          class="link"
+        >
+          samsung
+        </a>
+        <a
+          href="_blank"
+          class="link"
+        >
+          smartphones
+        </a>
+        <a
+          href="_blank"
+          class="link"
+          data-qa="hover"
+        >
+          laptops & computers
+        </a>
+        <a
+          href="_blank"
+          class="link"
+        >
+          gadegts
```

Comment:
@Yahnnis incorrect TYPO here. It should be `gadgets`

Reply:
ok